### DISCUSSION TITLE
How to have 2,4Ghz and 5,0Ghz network

### DISCUSSION THREAD

**Discussion by Ludo-code** [RaspAP/raspap-webgui#1526] [Q&A] [answered]
Hello,

how can i have 2 networks from the same network adaptater? Is the feature present in rasap?

**Comment by billz** [comment]
There's a dedicated doc page that discusses this https://docs.raspap.com/multiple/

**Reply by Ludo-code** [reply]
Yes I have seen :
Many users have asked if it's possible to create a second wireless access point on the same device. The answer is "yes" with an AP-capable external wireless adapter and the correct settings.


But I have wanted to know from the same adaptor, one that diffuses 5 and 2.4 and not one for 2.4 and another for 5ghz

**Reply by billz** [reply]
The configuration you describe is not supported.

**Reply by Ludo-code** [reply]
By rasap or in general?

**Reply by billz** [reply, marked as answer]
>  Is the feature present in rasap?

This feature is not present. Suggest you try experimenting with hostapd.conf

**Reply by Ludo-code** [reply]
Thank you I will try it

**Reply by billz** [reply]
This [related discussion](https://forum.openwrt.org/t/a-wireless-chip-containing-dual-band-2-4g-and-5g-only-generate-5g-configuration-in-openwrt-but-cannot-generate-2-4g-configuration/144428/10) on the OpenWRT forums could be useful.
